artemis-server/src/main/java/org/apache/activemq/artemis/core/paging/cursor/impl/PageSubscriptionImpl.java

```diff
@@ -860,6 +868,13 @@ public void processReload() throws Exception {
    public void stop() {
    }
 
+   @Override
+   public void counterSnapshot() {
+      if (counter != null) {
+         counter.snapshot();
+      }
```

Comment:
Seems odd for this method to null check when others such as isCounterPending() dont.